tests/integration/test_cubepart.py

```diff
@@ -318,25 +330,31 @@ def it_provides_values_for_cat_hs_x_mr(self):
     def it_provides_values_for_mean_cat_x_cat_hs(self):
         slice_ = Cube(CR.MEANS_CAT_X_CAT_HS).partitions[0]
 
-        np.testing.assert_array_almost_equal(
-            slice_.means,
-            np.array([[24.43935757, 37.32122746, np.nan, 55.48571956, 73.02427659]]),
-        )
-        np.testing.assert_array_almost_equal(slice_.rows_margin, np.array([np.nan]))
-        np.testing.assert_array_almost_equal(
-            slice_.columns_margin, np.array([np.nan] * len(slice_.counts[0, :]))
+        # This fixture has both cube_counts and cube_means measure, for this reason
+        # both measures are available at cubepart level.
+        assert slice_.means == pytest.approx(
+            np.array([[24.4393575, 37.3212274, np.nan, 55.4857195, 73.0242765]]),
+            nan_ok=True,
         )
+        assert slice_.counts == pytest.approx(np.array([[189, 395, 584, 606, 310]]))
```

Comment:
I think it's well to establish these asserts being in alphabetical order by property name, like all the other ones. It might seem like there is a logical ordering possible, now, while it's small, but as more property-value asserts are added that just comes undone. Better to be consistent with all the other tests like this from the start and then folks know what to expect and can find what they're looking for quickly.